x/cdp/types/params.go

```diff
@@ -39,22 +40,25 @@ var (
 	DefaultSurplusLot       = sdkmath.NewInt(10000000000)
 	DefaultDebtLot          = sdkmath.NewInt(10000000000)
 	stabilityFeeMax         = sdk.MustNewDecFromStr("1.000000051034942716") // 500% APR
+	// Run every block
+	DefaultBeginBlockerExecutionBlockInterval = int64(1)
 )
 
 // NewParams returns a new params object
 func NewParams(
 	debtLimit sdk.Coin, collateralParams CollateralParams, debtParam DebtParam, surplusThreshold,
-	surplusLot, debtThreshold, debtLot sdkmath.Int, breaker bool,
+	surplusLot, debtThreshold, debtLot sdkmath.Int, breaker bool, beginBlockerExecutionBlockInterval int64,
 ) Params {
 	return Params{
-		GlobalDebtLimit:         debtLimit,
-		CollateralParams:        collateralParams,
-		DebtParam:               debtParam,
-		SurplusAuctionThreshold: surplusThreshold,
-		SurplusAuctionLot:       surplusLot,
-		DebtAuctionThreshold:    debtThreshold,
-		DebtAuctionLot:          debtLot,
-		CircuitBreaker:          breaker,
+		GlobalDebtLimit:                    debtLimit,
+		CollateralParams:                   collateralParams,
+		DebtParam:                          debtParam,
+		SurplusAuctionThreshold:            surplusThreshold,
+		SurplusAuctionLot:                  surplusLot,
+		DebtAuctionThreshold:               debtThreshold,
+		DebtAuctionLot:                     debtLot,
+		CircuitBreaker:                     breaker,
+		BeginBlockerExecutionBlockInterval: beginBlockerExecutionBlockInterval,
```

Comment:
Maybe we should call this `LiquidationBlockInterval` now?  We aren't really skipping the begin blocker, just liquidations & those updates of risky cdps for the purpose of liquidation